services/movie_session.py

```diff
@@ -42,3 +42,10 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Using `get` without handling exceptions can lead to `DoesNotExist` errors if the session ID is not found. Consider handling this exception to avoid runtime errors.